### DISCUSSION TITLE
latest tesseract via home-brew depends on python 3.13

### DISCUSSION THREAD

**Discussion by mophilly** [enoch3712/ExtractThinker#172] [General]
I noticed that home-brew graciously installed python 3.13.1 when I installed tesseract. I understand that ExtractThinker is good on python 3.12.8. Lacking experience with tesseract I wonder if this presents a problem for local OCR tasks spawned by ExtractThinker.

**Comment by enoch3712** [comment]
Hello @mophilly! 

No, should be just fine. 

Tesseract is one of the best DocumentLoaders in the library, because i use it so much. 

https://enoch3712.github.io/ExtractThinker/core-concepts/document-loaders/tesseract/

You have plenty of examples here:
https://github.com/enoch3712/ExtractThinker/blob/main/tests/test_extractor.py
